uuid/validate_test.go

```diff
@@ -0,0 +1,42 @@
+package uuid
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestIsValid(t *testing.T) {
```

Comment:
For our test names we try to prefix them with what type of test they are. So in this case `TestUnit_IsValid`. We do this so we can filter tests when running them from our Makefiles. Like just trying to run Unit tests you can do `go test -r Unit` and then all the unit tests run. 
```suggestion
func TestUnit_IsValid(t *testing.T) {
```